### PR TITLE
Master legacy scss4 app

### PR DESCRIPTION
**Pull request by pparidans**
task-3439226

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/legacy/scss/modal.scss [hunk 1]
```diff
@@ -1,125 +1,28 @@
...
-                        background: $o-navbar-background;
-
-                        .modal-title, .o_subtitle, button.close {
-                            color: $o-navbar-entry-color;
-                        }
-                    }
+@include media-breakpoint-down(md) {
+    .modal {
+        .modal-content {
+            .modal-body.o_act_window .o_group.o_inner_group > tbody > tr > td:not(.o_td_label) {
```
[the remark this answers is not in the document]
Comment (pparidans) [reply addons/web/static/src/legacy/scss/modal.scss:19]: Or just... delete it actually

File: addons/web/static/src/legacy/scss/modal.scss
[the remark this answers is not in the document]
Comment (pparidans) [reply addons/web/static/src/legacy/scss/modal.scss:1]: Actually, yes... sadly... but not for long (looking at the legacy Dialog stuff...)

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Comment by adr-odoo** [comment]
@robodoo rebase-ff r+

**Review by adr-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**